optuna/integration/skorch.py

```diff
@@ -0,0 +1,44 @@
+from typing import Any
+
+import optuna
+
+with optuna._imports.try_import() as _imports:
+    from skorch.callbacks import Callback
+    from skorch.net import NeuralNet
+
+if not _imports.is_successful():
+    Callback = object  # NOQA
+
+
+class SkorchPruningCallback(Callback):
+    """Skorch callback to prune unpromising trials.
+
+    Args:
+        trial:
+            A :class:`~optuna.trial.Trial` corresponding to the current evaluation of the
+            objective function.
+        monitor:
+            An evaluation metric for pruning, e.g. ``val_loss`` or
+            ``val_acc``. The metrics are obtained from the returned dictionaries,
+            i.e., ``net.histroy``. The names are thus depend on how this dictionary
```

Comment:
A nit. Or, it may be a typo for `The names are thus dependent on ...`.
```suggestion
            i.e., ``net.histroy``. The names thus depend on how this dictionary
```

Reply:
Thanks.